api-reference/10 UI Components/dxTreeView/1 Configuration/dataStructure.md

```diff
@@ -8,7 +8,7 @@ default: 'tree'
 Notifies the UI component of the used data structure.
```

Comment:
I'd like to clarify "the used data structure". Are you talking about the data structure that is currently being used? If so, then would it be better to say:

`Notifies the UI component of the data structure in use.`